documentation/packages.xml

```diff
@@ -830,6 +830,9 @@ simple methods via GObject-Introspection
 		<package name="fwupd" gir="Fwupd-2.0" home="https://fwupd.org/" c-docs="https://fwupd.github.io/">
 			This project aims to make updating firmware on Linux automatic, safe and reliable.
 		</package>
+		<package name="libportal" home="https://github.com/flatpak/libportal" c-docs="https://flatpak.github.io/libportal/">
```

Comment:
You need to refer to the GIR file here to actually include its documentation. `gir="Xdp-1.0"`